app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -43,6 +46,27 @@ public static void setImageDrawable(
     imageView.setImageResource(drawableResourceId);
   }
 
+  /**
+   * Allows binding drawables to an [ImageView] via "android:src".
+   * Reference: https://stackoverflow.com/a/35809319/3689782.
+   */
+  @BindingAdapter("android:src")
+  public static void setImageDrawable(
```

Comment:
This is starting to be non-trivial. Could we add a simple test for the adapters to verify that the correct drawable is set per a particular ChapterPlayState?

Reply:
In `StoryActivityLocalTest` I have created following tests:
`testStoryFragment_completedChapter_checkProgressDrawableIsCorrect`
`testStoryFragment_notStartedChapter_checkProgressDrawableIsCorrect`
`testStoryFragment_lockedChapter_checkProgressDrawableIsCorrect`
`testStoryFragment_completedChapter_pawIconIsVisible`
`testStoryFragment_pendingChapter_pawIconIsGone`

Will tests suffice ?

Also, note that the file name is `StoryActivityLocalTest` (focus on Activity) so should I create a new file `StoryFragmentLocalTest` to add these tests? I think yes but still want to confirm if I am missing something.

Reply:
StoryFragmentTest sounds like a better idea (does it need to be a local test or can this also run on Espresso?).

That being said, I think we should also be introducing tests specifically for the adapters (to ensure they work correctly even when used in other locations). This is more in parity with our eventual policy to require each individual file to have a corresponding test file with 100% coverage for the production file. These tests will need to be added at some point, anyway, so adding them now seems easier since you have the others to reference.

WDYT?

Reply:
I have shifted all tests to `StoryFragmentTest` also introduced tests for bindable adapter too.